tools/serverpod_cli/lib/src/analyzer/dart/definitions.dart

```diff
@@ -55,6 +55,9 @@ class MethodDefinition {
   /// The named parameters of this method.
   final List<ParameterDefinition> parametersNamed;
 
+  /// Whether this method contains a stream input our output.
+  final bool isStream;
+
```

Comment:
Discussion: We are basically introducing a new method type here. What do you think about denoting this with an enum or even an entirely different class instead? 

We have a MethodStream(endpoint?) or a MethodCall(endpoint) it seem more extendable and clearer what the isStream = false means if we have a strict type instead.

Reply:
Makes sense! 👍